### PR TITLE
[ollama] Add better screenshots

### PR DESCRIPTION
**Pull request by mishig25**

### CODE REVIEW

**Review by julien-c — APPROVED** [review]

File: docs/hub/ollama.md [hunk 1]
```diff
@@ -6,6 +6,16 @@ Ollama is an application based on llama.cpp to interact with LLMs directly throu
 
 Getting started is as simple as:
 
+1. Enable `ollama` under your [Local Apps settings](https://huggingface.co/settings/local-apps).
+2. On a model page, choose `ollama` from `Use this model` dropdown. For example: [bartowski/Llama-3.2-1B-Instruct-GGUF](https://huggingface.co/bartowski/Llama-3.2-1B-Instruct-GGUF).
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/ollama-modelpage-light.png"/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/ollama-modelpage-dark.png"/>
```
Comment (pcuenca) [inline docs/hub/ollama.md:14]: They are gifs and not in `hub`: https://huggingface.co/datasets/huggingface/documentation-images/commit/85567fe3b60e8779876a2aa1f9386ae260acd090#d2h-192888

File: docs/hub/ollama.md [hunk 2]
```diff
@@ -25,14 +35,22 @@ ollama run hf.co/bartowski/Humanish-LLama3-8B-Instruct-GGUF
...
 
-To select a different scheme, simply add a tag:
+To select a different scheme, simply:
+
+1. From `Files and versions` tab on a model page, open GGUF viewer on a particular GGUF file.
+2. Choose `ollama` from `Use this model` dropdown.
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/ollama-modeltree-light.png"/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/ollama-modeltree-dark.png"/>
```
Comment (pcuenca) [inline docs/hub/ollama.md:45]: Same :)

File: docs/hub/ollama.md (hunk 1 above)
Comment (mishig25) [reply docs/hub/ollama.md]: fixed 😄

File: docs/hub/ollama.md (hunk 2 above)
Comment (mishig25) [reply docs/hub/ollama.md]: fixed 😄

**Pull request merged**